plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiUtil.java

```diff
@@ -39,7 +38,7 @@ public void testIsHudiParquetInputFormat()
         schema.setProperty(FILE_INPUT_FORMAT, HoodieParquetInputFormat.class.getName());
         schema.setProperty(SERIALIZATION_LIB, PARQUET.getSerde());
 
-        assertTrue(isHudiParquetInputFormat(getInputFormat(newEmptyConfiguration(), schema, false)));
+        assertTrue(getInputFormat(newEmptyConfiguration(), schema, false) instanceof HoodieParquetInputFormat);
```

Comment:
The better way to write this assertion is

```java
assertThat(getInputFormat(newEmptyConfiguration(), schema, false))
        .isInstanceOf(HoodieParquetInputFormat.class);
```

But we can simply delete this test method, as the point of is to test the `isHudiParquetInputFormat()` utility method, which no longer exists.